docs/environment-variables.md

```diff
@@ -19,11 +19,12 @@ export APP_MAIL_PASSWORD=<app-mail-password>
 
 | Environment Variable     | Description                                                                                                                                                                                                                    | Example |
 |--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 3 options for this: `dev`, `test` and `prod`.  To use the development environment configuration you should use "dev" as a value. | dev     |
+| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  To use the development environment configuration you should use "dev" as a value and If you want to use local database such as sqlite then set your environment config value as "local".| dev     |
```

Comment:
```suggestion
| FLASK_ENVIRONMENT_CONFIG | Short running environment name so that Flask know which configuration to load. Currently, there are 4 options for this: `dev`, `test`, `local` and `prod`.  If you want to use a simple local database (e.g.: sqlite) then set your environment config value as `local`. To use the development environment with a remote database configuration you should use `dev` as a value.  | dev     |
```